main.go

```diff
@@ -18,22 +18,38 @@ limitations under the License.
 package main
 
 import (
-	log "github.com/sirupsen/logrus"
+	"fmt"
 	"os"
+	"strconv"
 	"time"
 
+	log "github.com/sirupsen/logrus"
+
 	aggregator "github.com/nomad-node-problem-detector/aggregator"
 	config "github.com/nomad-node-problem-detector/config"
 	detector "github.com/nomad-node-problem-detector/detector"
 	"github.com/urfave/cli/v2"
 )
 
+var (
+	Timestamp string
+	GitCommit string
+)
+
 func main() {
+	// convert the timestamp to a date time to populate correctly the
+	// information for the app
+	i, err := strconv.ParseInt(Timestamp, 10, 64)
+	if err != nil {
+		panic(fmt.Sprintf("failed to convert %s to an integer: %v", Timestamp, err))
+	}
+	compiledAt := time.Unix(i, 0)
+
 	app := &cli.App{
 		Name:                 "npd",
 		Usage:                "Nomad node problem detector",
-		Version:              "v1.0.0",
-		Compiled:             time.Now(),
+		Version:              GitCommit,
```

Comment:
Shall we include the version number along with Git commit? Although the version number is a bit outdated `v1.0.0`, I initially wanted it to mirror the release version tag e.g. [`v0.5`](https://github.com/Roblox/nomad-node-problem-detector/releases/tag/v0.5)

So something like:

```
$ ./npd
NAME:
   npd - Nomad node problem detector

USAGE:
   npd [global options] command [command options] [arguments...]

VERSION:
   v0.5 a08e2c16d59e12e61cf262711c486bc480edc547
```

```
$ ./npd --version
npd version v0.5 a08e2c16d59e12e61cf262711c486bc480edc547
```

Reply:
Yes, good idea, I'll add the tag.

Reply:
Thanks. Let me check why CI is failing.